portfolio/src/main/webapp/script.js

```diff
@@ -44,3 +44,8 @@ function addFact() {
 
 }
 
+async function getHelloWorldText() {
+  const response = await fetch('/data');
+  const hello_text = await response.text();
```

Comment:
It's more common in JavaScript to use camelCase naming instead of snake_case.

Reply:
Gotcha, fixed this and other occurrences of the same in later PRs